srcpkgs/torbrowser-launcher/patches/wayland.diff

```diff
@@ -0,0 +1,14 @@
+diff --git a/apparmor/torbrowser.Browser.firefox b/apparmor/torbrowser.Browser.firefox
+index 6882f9d..365b073 100644
+--- a/apparmor/torbrowser.Browser.firefox
++++ b/apparmor/torbrowser.Browser.firefox
+@@ -115,6 +115,9 @@ profile torbrowser_firefox @{torbrowser_firefox_executable} {
+   owner /{dev,run}/shm/org.chromium.* rw,
+   owner /dev/shm/org.mozilla.ipc.[0-9]*.[0-9]* rw, # for Chromium IPC
+ 
++  # Required for Wayland display protocol support
++  owner /dev/shm/wayland.mozilla.ipc.[0-9]* rw,
```

Comment:
`abstractions/wayland` exists, I think we could include that directly.

Reply:
Doesn't [seem](https://gitlab.com/apparmor/apparmor/-/blob/master/profiles/apparmor.d/abstractions/wayland) to include the right socket?

Reply:
You're right, thanks. It seems `abstractions/gnome` pulls in `wayland` already.